spec/features/users/06_user_edits_account_info_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rails_helper"
+
+# Acceptance Criteria:
+# [x] user can see current account details page
+# [x] user can navigate to edit account page
+# [x] user can update name
+# [x] user can update email
+# [x] user can update profile picture
+# [x] user can update password
+# [x] user must enter current password for update to persist
+
+feature "edit account details" do
+
+  let (:user) { FactoryGirl.create(:user) }
+  let! (:lifehack) { FactoryGirl.create(:lifehack, creator: user) }
+  let! (:review) do
+    FactoryGirl.create(:review, creator: user, lifehack: lifehack)
+  end
+
+  scenario "user sees current account details" do
+    user_sign_in(user)
+    click_link "Logged In As: #{user.first_name}"
+
+    expect(page).to have_content("User Details")
+    expect(page).to have_content("Name: #{user.full_name}")
+    expect(page).to have_content("Email: #{user.email}")
+    expect(page).to have_content(lifehack.title)
+    expect(page).to have_content(review.lifehack.title)
+  end
+
+  scenario 'user navigates to edit account page' do
+    user_sign_in(user)
+    click_link "Logged In As: #{user.first_name}"
+    click_link "Update Account Information"
+  end
+
+  scenario "user must enter current password for profile update" do
+    user_sign_in(user)
```

Comment:
Might be worth factoring `user_sign_in(user)` into a `before` block for this set of tests.